### ISSUE TITLE
Integration Test Failure climada_petals.test.test_cropyield_integr.TestIntegr.test_EU

### ISSUE THREAD

**Issue by emanuel-schmid** [CLIMADA-project/climada_petals#11]
**Error Message**: `52278210.72839116 != 53074789.755290434 within 7 places (796579.0268992707 difference)`

**Stacktrace**:
```
Traceback (most recent call last):
  File "/var/lib/jenkins/jobs/petals_ci_night/workspace/climada_petals/test/test_cropyield_integr.py", line 77, in test_EU
    self.assertAlmostEqual(exp.gdf.value.max(), 53074789.755290434)
AssertionError: 52278210.72839116 != 53074789.755290434 within 7 places (796579.0268992707 difference)
```

Before we just change the expected value, the cause for this sudden change should be clarified.

**Comment by sameberenz** [comment]
Hi Emanuel, is there a way to tell since when / after which commit to which branch this error has occured?

**Comment by emanuel-schmid** [comment]
Yes, of course. You just look at the outcome of the nightly integration tests on jenkins and you'll find that the failure first occured here: https://ied-wcr-jenkins.ethz.ch/job/petals_ci_night/71 which tells you that the one suspicious commit is https://github.com/CLIMADA-project/climada_petals/commit/884527ba4b193de249b7b266e414c5dd6c46474c. 
However - it's also possible that the discrepancy doesn't root in calculation but in resources, be it data from elsewhere or changed dependency versions

**Comment by sameberenz** [comment]
Thanks, it seems like a change in the assignment of countries (region_id) has changed. This can explain the change in accumulated results per country and is plausible. I will look into it and update the test.

**Comment by sameberenz** [comment]
Since the integration test is currently failing on a highler level due to the issue described in https://github.com/CLIMADA-project/climada_petals/issues/15 , I suggest to postpone fixing this bug untill https://github.com/CLIMADA-project/climada_petals/issues/15 is merged.

Once this is done, we can run the integration test `test_cropyield_integr` again and change the test values because the reason is the change in region_id assignment as described above.

**Comment by emanuel-schmid** [comment]
Sorry, but I cannot fully grasp the reasoning. Are you worried about merging conflicts and you prefer to first merge #12 (?) before tackling this one?

**Comment by sameberenz** [comment]
The test does not run through due to #12, so I can not further inquire on other errors as long as #12 is not solved.

**Comment by emanuel-schmid** [comment]
#12 _per se_ is solved.

**Comment by sameberenz** [comment]
solved in https://github.com/CLIMADA-project/climada_petals/pull/12 on branch  feature/deprecate_set_cropprod
